lib/cron/cache-pools-ipfs.ts

```diff
@@ -0,0 +1,112 @@
+import pinataSDK from '@pinata/sdk';
+import { ChainId, SubgraphProvider } from '@uniswap/smart-order-router';
+import { EventBridgeEvent, ScheduledHandler } from 'aws-lambda';
+import { Route53, STS } from 'aws-sdk';
+import { default as bunyan, default as Logger } from 'bunyan';
+import fs from 'fs';
+
+// if (process.env.STAGE == 'beta') {}
+
+const DIRECTORY = '/tmp/pools/';
```

Comment:
Can we get the v1 in here? `/tmp/v1/pools`? then we pin `v1` directory?

Reply:
Do you mean v3? Why v1?

Reply:
Oh yeah sorry this was a confusing comment. Our API is versioned, so the current quote endpoint is api.uniswap.org/v1/quote where v1 means version 1 of the API. 

So to fit `pools` endpoint into that structure we would want to create a folder 'v1' on ipfs. Then our final IPNS urls would be like: `ipfs.io/ipns/api.uniswap.org/v1/pools/mainnet/v3`

Reply:
oh I see. The only issue is that the there needs to be a file that we write to. So we could do something like:
`ipfs.io/ipns/api.uniswap.org/v1/pools/v3/mainnet.txt`. Or open to other suggestions, just need a .txt or .json